### PR TITLE
[hotfix][flink-streaming-java] modify typo in the StreamingJobGraphGenerator.java

### PR DESCRIPTION
**Pull request by maqingxiang**

### CODE REVIEW

**Comment by yanghua** [comment]
+1, LGTM cc @tillrohrmann @zentol

File: flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java [hunk 1]
```diff
@@ -130,7 +130,7 @@ private JobGraph createJobGraph() {
 		jobGraph.setScheduleMode(ScheduleMode.EAGER);
 
 		// Generate deterministic hashes for the nodes in order to identify them across
-		// submission iff they didn't change.
+		// submission if they didn't change.
```
Comment (Aitozi) [inline flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java]: I think here means `if and only if` and is not a typo.

Comment (yanghua) [reply flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java]: There are indeed abbreviations of this form, and there is no problem with this understanding. Agree.

Comment (GJL) [reply flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java]: I am reverting this change.

**Comment by GJL** [comment]
LGTM, merging.

**Pull request closed without merging**